app/tests/spec/views/complete_reset_password.js

```diff
@@ -361,9 +364,12 @@ function (chai, sinon, p, AuthErrors, Metrics, FxaClient, InterTabChannel,
                   EMAIL,
                   PASSWORD,
                   relier,
+                  // TODO: remove once https://github.com/eslint/eslint/issues/1801 gets fixed
+                  /* eslint-disable */
```

Comment:
A couple other workarounds:
```js
      return self.fxaClient.signIn(
          email,
          password,
          self.relier,
          { reason: self.fxaClient.SIGNIN_REASON.ACCOUNT_UNLOCK }
        )
```

Or:
```js
      var reason = {
        reason: self.fxaClient.SIGNIN_REASON.ACCOUNT_UNLOCK
      };
      return self.fxaClient.signIn(
          email,
          password,
          self.relier,
          reason
        )
```

Meh. Not a huge fan of hacking code just to pass the linters though.